app/extensions/brave/content/scripts/idleHandler.js

```diff
@@ -1,5 +1,6 @@
 chrome.idle.setDetectionInterval(15 * 60)
 chrome.idle.onStateChanged.addListener((idleState) => {
+  // uses appConstants.APP_IDLE_STATE_CHANGED action constant
```

Comment:
nit: start comment with capital and end in period.

Reply:
done